tests/integration/run.bats

```diff
@@ -77,6 +77,19 @@ function teardown() {
 	[[ "${lines[0]}" == *'mydomainname'* ]]
 }
 
+@test "runc run with tmpfs" {
+	requires root
+
+	mode=$(stat -c %A /tmp)
```

Comment:
I'm not sure this test is actually testing the fix directly -- you want `rootfs/tmp` to have the same mode even after we mount tmpfs on top, not necessarily the same mode as the host `/tmp`. In addition, I would suggest setting the sticky bit on `rootfs/tmp` explicitly so it's clearer what is being tested. Maybe something like

```suggestion
	chmod 'a=rwx,ug+s,+t' rootfs/tmp # set all bits
	mode=$(stat -c %A rootfs/tmp)
```

Reply:
Indeed, I was careless just now.
Thanks.